ftw/testing/freezer.py

```diff
@@ -48,6 +48,8 @@ def __enter__(self):
         def freezed_now(klass, tz=None):
             if not tz:
                 return self.new_now.replace(tzinfo=None)
+            elif self.new_now.tzinfo is None:
```

Comment:
I would love a comment here that at least somewhat describes what's happening in this edge-case. Something like
```
# Time was frozen to a naive DT, but a TZ-aware time is being requested from now()
```